brayns/common/types.h

```diff
@@ -335,6 +340,9 @@ enum class CameraMode
     inspect
 };
 
+// A clip plane is defined by a normal and a distance to the center of the scene bounding box
+typedef std::vector< Vector4f > ClipPlanes;
```

Comment:
The 4 floats contain absolute values, so changing the size of the scene does not affect the equation, and clip planes are reset anyway whenever the scene is changed. I'll update the documentation accordingly.